etc/db-config.yaml

```diff
@@ -98,10 +98,14 @@
             public: false
             description: Maximum size of run/diff/error/system output shown in the jury interface (in bytes); use "-1" to disable any limits.
         -   name: lazy_eval_results
-            type: bool
-            default_value: true
+            type: int
+            default_value: 0
```

Comment:
we need a DB migration for this

Reply:
Don’t we store bools as int already? If so we don’t